### PR TITLE
Fix preview text

### LINKED ISSUE #8323: Message previews show in Html format
### Steps to reproduce

Not much investigation done yet, doesn't look really deterministic for now but might be caused by #8235

### Expected behavior

message previews should be in text-only format, the same as when you open the message 

### Actual behavior

sometimes message preview contains HTML syntax

image

### Mail app version

3.1.0-alpha.1

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by hamza221**
ref #8222 
fixes  #8323

bugs fixed

- [x] Image alt removed 
- [x] Links rendered as` [Text](https://example.com) ` are now simply => Text
- [x] Message previews show in HTML format

### CODE REVIEW

**Comment by hamza221** [comment]
Checking for textBody first might cause  https://github.com/nextcloud/mail/issues/8323 and 


Image

**Comment by hamza221** [comment]
Yes here  is problematic example from gmail
image

**Comment by hamza221** [comment]
Failing test looks related to #8694 @JohannesGGE

**Comment by ChristophWurst** [comment]
> Failing test looks related to #8694

I think it's from https://github.com/nextcloud/mail/pull/8752

**Comment by JohannesGGE** [comment]
Fixed it in https://github.com/nextcloud/mail/pull/8759
Rebase schould fix it

File: composer.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		"christophwurst/kitinerary-sys": "^0.2.1",
 		"ezyang/htmlpurifier": "4.16.0",
 		"gravatarphp/gravatar": "dev-master#6b9f6a45477ce48285738d9d0c3f0dbf97abe263",
-		"html2text/html2text": "^4.3.1",
+		"hamza221/html2text": "^1.0",
```
Comment (miaulalala) [inline composer.json:35]: Did you have to fork your own version? :scream_cat:

**Review by miaulalala — APPROVED** [review]
LGTM

File: composer.json (hunk 1 above)
Comment (hamza221) [reply composer.json:35]: yeah, I have a Pr open since march 
https://github.com/mtibben/html2text/pull/116

**Pull request merged**